scss/components/_reveal.scss

```diff
@@ -175,4 +169,12 @@ $reveal-overlay-background: rgba($black, 0.45) !default;
       position: fixed;
     }
   }
+
+  @include breakpoint(medium) {
+    %reveal-centered {
+      right: auto;
+      left: auto;
+      margin: 0 auto;
+    }
+  }
```

Comment:
Introduce a bug when `reveal-modal-width` is called without `foundation-reveal`.